hass-mixin/dashboard.libsonnet

```diff
@@ -5,6 +5,7 @@ local base_matcher = 'job=~"$job", instance=~"$instance"';
 local entity_matcher = base_matcher + ', entity=~"$entity", friendly_name=~"$friendly_name"';
 local available_entity_matcher_decoration = 'and on (entity) entity_available > 0';
 
+local prefix_default = 'homeassistant';
```

Comment:
Odd.. I thought I had already enabled this. I suppose when I built/tested I was always using a custom prefix.

Thanks for this contribution!